hero.py

```diff
@@ -0,0 +1,108 @@
+from monster import Monster
+
+
+class Hero:
+    # The HP percents that the hero wil gain back - 50%
+    HEALING_PERCENTS = 1.5
+    # The damage points that the hero wil gain back - 20%
+    DAMAGE_PERCENTS = 1.5
+    # The power by which the coin level grows in order to level up- times 3
+    UPGRADE_COIN_POWER = 2
+    # Reduces damage when hero chooses to defend- 80%
+    PROTECTION_PERCENTS = 0.8
```

Comment:
this is fine, but officially these class fields are supposed to be a num of percent